src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Remote/RemoteClientInitializationOptions.cs

```diff
@@ -37,6 +37,9 @@ internal struct RemoteClientInitializationOptions
     [JsonPropertyName("useNewFormattingEngine")]
     public required bool UseNewFormattingEngine { get; set; }
 
-    [JsonPropertyName("avoidExplicitCommitCharacters")]
+    [JsonPropertyName("supportsSoftSelectionInCompletion")]
     public required bool SupportsSoftSelectionInCompletion { get; set; }
+
+    [JsonPropertyName("vsCodeCompletionTriggerCharacters")]
```

Comment:
Since you have to make at least one more commit for the failing test anyway, I'm going to be annoying: Nit, should be "useVSCodeCompletionTriggerCharacters" 😛